Source/WebKit/Shared/WebCoreArgumentCoders.cpp

```diff
@@ -344,58 +344,6 @@ std::optional<Ref<FontCustomPlatformData>> ArgumentCoder<FontCustomPlatformData>
     return fontCustomPlatformData.releaseNonNull();
 }
 
-void ArgumentCoder<WebCore::FontPlatformData::Attributes>::encode(Encoder& encoder, const WebCore::FontPlatformData::Attributes& data)
```

Comment:
You need to leave this (and the other handwritten coders) in place for all the non-cocoa platforms to keep their build working.

A `#if !USE(CORE_TEXT)` should work.